### PR TITLE
different types of alert sections made

### PR DESCRIPTION
**Pull request by Asypher00**
Screenshot (3)

### CODE REVIEW

File: alert.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.alert-container{
+
```
Comment (tanishq20) [inline alert.css]: Avoid giving extra space.

File: alert.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    font-size: 1.25rem;
+    font-family: "comic sans ms";
+    width : 50vw;
+    border-radius : 0.625rem;
+    justify-content: space-between;
+    margin : 1.25rem;
+}
+
+.alert-container-error{
+    background-color: rgb(245, 125, 125);
```
Comment (tanishq20) [inline alert.css]: It will be good if you pre-define these colors in root and import them as var().

File: alert.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/228add9a8b.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="alert.css">
+    <title>Document</title>
```
Comment (Sameony) [inline alert.html]: Change the title of the document to alert

File: alert.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            This is a success message</div>
+            <div class="alert-footer"><button class="alert-btn alert-container-success"><i class="far fa-times-circle"></i></button></div>
+    </div>
+    <div class="alert-container alert-container-warning ">
+        <div class="alert-header">
+            <i class="fas fa-exclamation-triangle"></i>
+            This is a warning message</div>
+            <div class="alert-footer"><button class="alert-btn alert-container-warning"><i class="far fa-times-circle"></i></button></div>
+    </div>
+
```
Comment (Sameony) [inline alert.html:30]: Format the code using beautify extension for better readability

File: alert.css (hunk 1 above)
Comment (Asypher00) [reply alert.css]: done! Thanks for the review

File: alert.css (hunk 2 above)
Comment (Asypher00) [reply alert.css]: done! Thanks for the review

File: alert.html (hunk 3 above)
Comment (Asypher00) [reply alert.html]: done! Thanks for the review

**Pull request merged**